jquery.pjax.js

```diff
@@ -118,15 +118,16 @@ function handleClick(event, container, options) {
 function handleSubmit(event, container, options) {
   options = optionsFor(container, options)
 
-  var form = event.currentTarget
+  var form = event.currentTarget;
+  var $form = $(form);
 
   if (form.tagName.toUpperCase() !== 'FORM')
     throw "$.pjax.submit requires a form element"
 
   var defaults = {
     type: form.method.toUpperCase(),
```

Comment:
It might be good that you change it. `$(form).attr('method') || 'GET'` should be equivalent to `form.method`